samples/GiftedChatReactNativeControl/android/app/src/main/java/com/giftedchatreactnativecontrol/MainActivity.java

```diff
@@ -0,0 +1,8 @@
+package com.giftedchatreactnativecontrol;
+
+import com.reactnativenavigation.NavigationActivity;
+
+public class MainActivity extends NavigationActivity {
```

Comment:
I think we are using this class because this is the point of entry?

Reply:
Yes, and all the .java files were auto-generated by react-native.